### PR TITLE
#166272095 setup test environment

### PR DESCRIPTION
**Pull request by frankly034**
#### What does this PR do?
set up the test environment with mochai, chai and chai-http

#### Description of Task to be completed?
- install and setup babel 
- install test dev-dependencies: mochai, chai, chai-http
- create default test

#### How should this be manually tested?
After cloning the repo, CD into it and run ```npm run test``` 

#### Any background context you want to provide?
This is to setup a basic test environment

#### What are the relevant pivotal tracker stories?
[Delivers #166272095]

### CODE REVIEW

**Review by Sojisoyoye — APPROVED** [review]
All necessary test dependencies installed and tested. Good job Lewis

**Review by akinyeleolat — COMMENTED** [review]
Well done Lewis, The setup looks good to me but at what point will database for testing be integrated.

**Comment by frankly034** [comment]
> Well done Lewis, The setup looks good to me but at what point will database for testing be integrated.

It would be set on the Environment Variable, I have added the environment set to test at point of running the test in the test script. 

Thanks for your review

**Review by EmmanuelSage — APPROVED** [review]
Good job Lewis,
it looks okay to me.

File: test/default.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+chai.use(chaiHttp);
+
```
Comment (benfluleck) [inline test/default.spec.js]: Change this filename to `index.spec.js`

**Review by benfluleck — CHANGES_REQUESTED** [review]
I have left a comment

File: test/default.spec.js (hunk 1 above)
Comment (frankly034) [reply test/default.spec.js]: Ok. I have done that.

**Review by benfluleck — APPROVED** [review]

**Pull request merged**